src/main/java/duke/Parser.java

```diff
@@ -18,15 +20,15 @@ else if (str.split(" ")[0].equals("unmark")) {
                     taskList.unmarkTask(str, taskList);
                     storage.writeToFile(taskList);
                 }
-                else if(str.split(" ", 2)[0].equals("todo")) {
+                else if(str.split(" ", 2)[0].equals("duke.todo")) {
```

Comment:
Some minor improvement suggestions (feel free to use as many or as little as you'd like xd)
1.  Perhaps give `str.split(" " ,2)[0] `  a better name? Something like `header`
2.  Maybe can try using a switch statement if you're just doing comparisons?